app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -124,12 +124,22 @@ def analyze
                           msg % [options[:image_full_name], options[:docker_image_id][0..11], actual[0..11]],
                           'error')
     end
+
+    collect_compliance_data(image)
+
     image.scan_metadata(SCAN_CATEGORIES,
                         "taskid" => jobid,
                         "host"   => MiqServer.my_server,
                         "args"   => [YAML.dump(scan_args)])
   end
 
+  def collect_compliance_data(image)
+    # openscap_result = OpenscapResult.new(:container_image => image)
```

Comment:
Yes. The intention is to remove the comments when a new image_inspector_client is available. See also [this](https://github.com/ManageIQ/manageiq/pull/6839#issuecomment-200556024) comment